src/oge/reference_tables/epa_eia_crosswalk_manual.csv

```diff
@@ -190,4 +190,66 @@ plant_id_epa,emissions_unit_id_epa,plant_id_eia,generator_id,start_year,end_year
+65373,CT-6,65373,CTG-6,,,Matched based on generator ID
+1702,A,1702,3,,,Matched based on fuel type and fuel consumption
+1702,B,1702,4,,,Matched based on fuel type and fuel consumption
+2503,BLR114,2503,GT1,,,Only one generator at this plant
+2503,BLR115,2503,GT1,,,Only one generator at this plant
+2503,BLR116,2503,GT1,,,Only one generator at this plant
+2503,BLR117,2503,GT1,,,Only one generator at this plant
+2503,BLR118,2503,GT1,,,Only one generator at this plant
+2504,120,2504,GT1,,,Should be excluded according to CAMD
+2504,121,2504,GT1,,,Should be excluded according to CAMD
+2504,122,2504,GT1,,,Should be excluded according to CAMD
+2828,B008,2828,1,,,Matched based on fuel consumption
+2828,B010,2828,2,,,Matched based on fuel consumption
+3399,A1,3399,1,,,Matched based on generator ID
+3399,B1,3399,1,,,Matched based on generator ID
+3935,AUX1,3935,1,,,Matched based on generator ID
+3935,AUX2,3935,1,,,Matched based on generator ID
+3948,AUX1,3948,1,,,Matched based on generator ID
+6166,AB1,6166,MB1,,,Matched based on generator ID
+6166,AB2,6166,MB2,,,Matched based on generator ID
+6264,AUX1,6264,1,,,Matched to unique generator ID at the plant
+6264,AUX2,6264,1,,,Matched to unique generator ID at the plant
+8102,B001,8102,1,,,Matched based on generator ID
+8102,B002,8102,1,,,Matched based on generator ID
+10856,GB1,10865,GEN7,,,Matched based on fuel consumption
+10856,GB2,10865,GEN8,,,Matched based on fuel consumption
+10866,13,10866,GEN6,,,Matched based on opreation date
+10866,14,10866,GEN7,,,Matched based on opreation date
+50240,2,50240,GEN2,,,Unclear. Both GEN1 and GEN2 belong to same subplant
+50240,3,50240,GEN1,,,Unclear.
+50240,5,50240,GEN1,,,Unclear.
+50240,7,50240,GEN1,,,Unclear.
+50472,BLR05,50472,GEN1,,,Unclear. Both generator at plant have 4 boilers named BLR01 -> BLR04
+50479,6,50479,GEN1,,,Matched based on energy source code of generator
+50481,253-28,50481,TG21,,,Generators TG16 to TG21 are associated to boiler 253-25 to 253-29. TG21 is the only generator with no CAMD unit
+50729,CLBLR1,50729,GEN3,,,Matched based on boiler ID and fuel consumption
+50729,CLBLR2,50729,GEN3,,,Matched based on boiler ID and fuel consumption
+50733,701B1,50733,STG1,,,Only one generator at this plant
+50733,701B2,50733,STG1,,,Only one generator at this plant
+50733,701B3,50733,STG1,,,Only one generator at this plant
+50733,701B5,50733,STG1,,,Only one generator at this plant
+50733,701B6,50733,STG1,,,Only one generator at this plant
+50733,720B1,50733,STG1,,,Only one generator at this plant
+50733,720B2,50733,STG1,,,Only one generator at this plant
+50733,720B3,50733,STG1,,,Only one generator at this plant
+50900,5,50900,GEN3,,,Matched based on fuel consumption
+50900,11,50900,GEN3,,,Matched based on fuel consumption
+50900,1,50900,GEN3,,,Matched based on fuel consumption
+50900,4,50900,GEN3,,,Matched based on fuel consumption
+52089,BLR010,52089,GEN4,,,Matched based on fuel consumption. All generators at plant share same boilers and stack

```

Comment:
According to the BGA table it looks like these units map to boilers of the same name in EIA, and each of those boilers maps to all generators 1-4 at this plant.

Reply:
Ok. I will map all for CAMD units to all four generators.

Reply:
I'll add this to my PR